catalog-rest-service/src/main/resources/ui/src/constants/constants.ts

```diff
@@ -48,6 +48,21 @@ export const tiers = [
   { key: 'Tier.Tier5', doc_count: 0 },
 ];
 
+export const sortingFields = [
+  { name: 'Weekly_stats', value: 'weekly_stats' },
```

Comment:
Name should not be key, instead should be human-readable text.
`Weekly Stats`, `Daily Stats`, `Monthly Stats` and `Last Updated Timestamp`